src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/DefaultTranscriptsList.tsx

```diff
@@ -93,6 +93,7 @@ const DefaultTranscriptslist = (props: Props) => {
       <div className={styles.header}>
         {isFilterOpen && (
           <TranscriptsFilter
+            shouldShowFilterIndicator={shouldShowFilterIndicator}
```

Comment:
😲 why? TranscriptsFilter already contains all the information about the current sorting rule and the selected filters.

Reply:
Just wanted to avoid code duplication

Reply:
We may have something like the following:

In the `DefaultTranscriptsList` component:

```tsx
import classNames from 'classnames'

// other code

const filterSortBoxLabel = (
  <span
    className={classNames({[transcriptListStyles.labelWithActivityIndicator]: shouldShowFilterIndicator })
  >
    Filter & sort
  </span>
);

// then where you use this label in DefaultTranscriptsList:

{ gene.transcripts.length > 5 && !isFilterOpen && filterSortBoxLabel }

// and for TranscriptsFilter component

<TranscriptsFilter label={filterSortBoxLabel} ... />

// in which case the props of TranscriptsFilter component should include
// { label: ReactNode }
```

Reply:
✅